pkg/kubecost/allocation.go

```diff
@@ -108,6 +107,13 @@ type RawAllocationOnlyData struct {
 	RAMBytesUsageMax float64 `json:"ramByteUsageMax"`
 }
 
+// PVUsage contains the byte hour usage
+// and cost of an Allocation for a single PV
+type PVUsage struct {
```

Comment:
This name I'd also like to think about. "Usage" has a loaded meaning in Kubecost; namely, a resource "allocation" is the max of "usage" and "request." I wonder if we could get away with just calling this `PV` or `PVC` or perhaps `PVAllocation`? I'm foreseeing us potentially looking to expand what we store in this, so a more general name might age better.

Also, should we keep the name on the object as `Name string`? For the purpose of this PR, you have name on the `PVBreakdown` map, but that makes individual instances of this type less useful stand-alone.